### DISCUSSION TITLE
Mass selection of items

### DISCUSSION THREAD

**Discussion by akidone** [jonaswinkler/paperless-ng#1356] [Feature Requests]
I wonder if there is any chance to get a mass selection checkbox for the search dialogue?

Currently I need to alter that tags on around 600+ entries, and I need to manually select each one of them.

I guess this could be very handy for many people - what do you say?

Anyways, thanks for paperless-ng, I helps me organizing my life a bit better ;-)

**Comment by sbrunner** [comment]
It already exists... =>
- Do the search
- Select one item
- Click on All in the toolbar that just replace the search bar...

**Comment by akidone** [comment]
Oh man. Sometimes the answer is so simple, RTFM ftw. Thanks for helping me!
